typescript/cli/src/config/strategy.ts

```diff
@@ -0,0 +1,186 @@
+import { confirm, input, password, select } from '@inquirer/prompts';
+import { Wallet } from 'ethers';
+import { stringify as yamlStringify } from 'yaml';
+
+import {
+  ChainSubmissionStrategy,
+  ChainSubmissionStrategySchema,
+  TxSubmitterType,
+} from '@hyperlane-xyz/sdk';
+import {
+  ProtocolType,
+  assert,
+  errorToString,
+  isAddress,
+  isPrivateKeyEvm,
+} from '@hyperlane-xyz/utils';
+
+import { CommandContext } from '../context/types.js';
+import { errorRed, log, logBlue, logGreen, logRed } from '../logger.js';
+import { runSingleChainSelectionStep } from '../utils/chains.js';
+import {
+  indentYamlOrJson,
+  isFile,
+  readYamlOrJson,
+  writeYamlOrJson,
+} from '../utils/files.js';
+import { maskSensitiveData } from '../utils/output.js';
+
+/**
+ * Reads and validates a chain submission strategy configuration from a file
+ */
+export async function readChainSubmissionStrategyConfig(
+  filePath: string,
+): Promise<ChainSubmissionStrategy> {
+  log(`Reading submission strategy in ${filePath}`);
+  try {
+    const strategyConfig = readYamlOrJson<ChainSubmissionStrategy>(filePath);
+
+    const parseResult = ChainSubmissionStrategySchema.parse(strategyConfig);
+
+    return parseResult;
+  } catch (error) {
+    logRed(`⛔️ Error reading strategy config:`, errorToString(error));
+    throw error; // Re-throw to let caller handle the error
+  }
+}
+
+/**
+ * Safely reads chain submission strategy config, returns empty object if any errors occur
+ */
+export async function safeReadChainSubmissionStrategyConfig(
+  filePath: string,
+): Promise<ChainSubmissionStrategy> {
+  try {
+    const trimmedFilePath = filePath.trim();
+    if (!isFile(trimmedFilePath)) {
+      logBlue(`File ${trimmedFilePath} does not exist, returning empty config`);
+      return {};
+    }
+    return await readChainSubmissionStrategyConfig(trimmedFilePath);
+  } catch (error) {
+    logRed(
+      `Failed to read strategy config, defaulting to empty config:`,
+      errorToString(error),
+    );
+    return {};
+  }
+}
+
+export async function createStrategyConfig({
+  context,
+  outPath,
+}: {
+  context: CommandContext;
+  outPath: string;
+}) {
+  let strategy: ChainSubmissionStrategy;
+  try {
+    const strategyObj = await readYamlOrJson(outPath);
+    strategy = ChainSubmissionStrategySchema.parse(strategyObj);
+  } catch (_) {
```

Comment:
if you're not going to use the err var, don't include the parens. 
So simply: ` } catch {`